### PR TITLE
Kkraune/bool int attr MERGEOK

### PR DESCRIPTION
**Pull request by kkraune**
@geirst the important part is to review the memory calculations, seems like same as for other numeric

@baldersheim @bratseth FYI

### CODE REVIEW

File: documentation/performance/attribute-memory-usage.html [hunk 1]
```diff
@@ -267,6 +259,16 @@ <h3 id="variants">Variants</h3>
       <td>Document vector, Multi-value mapping, Enum store, Posting list</td>
       <td>D * IW * ROF + D * V * EW * ROF + U * (FW+VW+4) * ROF + D * V * PW * ROF</td>
     </tr>
+    <tr>
+      <td>Boolean single-value</td>
+      <td>Document vector</td>
+      <td>D * FW * ROF</td>
+    </tr>
+    <tr>
+      <td>Boolean multi-value</td>
```
Comment (geirst) [inline documentation/performance/attribute-memory-usage.html]: I think this still should be "Numeric multi-value value plain"?

File: documentation/performance/feature-tuning.html [hunk 2]
```diff
@@ -119,4 +119,11 @@ <h2 id="document-summaries">Document summaries</h2>
 
 <h2 id="boolean-numeric-text-attribute">Boolean, numeric, text attribute</h2>
 <p>
+When selecting attribute field type, considering performance, this is a rule of thumb:
+<ol>
+  <li>Use boolean if a field is a boolean (max two values)</li>
+  <li>Use a string attribute if there is a set of values - only unique attributes are stored</li>
```
Comment (geirst) [inline documentation/performance/feature-tuning.html]: Change to "only unique strings"?

File: documentation/performance/attribute-memory-usage.html (hunk 1 above)
Comment (kkraune) [reply documentation/performance/attribute-memory-usage.html]: not sure what you mean? I added bool single / multi value - and removed a duplicate entry for _Numeric multi-value value plain_ at (see https://docs.vespa.ai/documentation/performance/attribute-memory-usage.html)

File: documentation/performance/feature-tuning.html (hunk 2 above)
Comment (kkraune) [reply documentation/performance/feature-tuning.html]: willdo!

**Comment by geirst** [comment]
👍

**Pull request merged**